.github/workflows/build-rust-code.yml

```diff
@@ -72,10 +72,9 @@ jobs:
         persist-credentials: false
 
     - name: Rust install
-      uses: actions-rs/toolchain@v1
+      uses: dtolnay/rust-toolchain@master
```

Comment:
should we point to stable here too rather than master?

Reply:
I guess it will depend if you want to use a specific version of the go tool chain or not.
According to the documentation about the tool chain parameter :
> Rustup toolchain specifier e.g. stable, nightly, 1.42.0, nightly-2022-01-01. Important: the default is to match the @rev as described above. When passing an explicit toolchain as an input instead of @rev, you'll want to use "dtolnay/rust-toolchain@master" as the revision of the action.

Reply:
@jbpaux thanks. Sounds like we can have them both be master then and should switch `check-rust.yml` to `@master`.

Reply:
exact. I don't know why I used `@stable` for `check-rust.yml`. I'll update.